src/shogun/statistical_testing/QuadraticTimeMMD.cpp

```diff
@@ -66,7 +67,7 @@ struct CQuadraticTimeMMD::Self
 	SGVector<float64_t> gamma_fit_null();
 
 	CQuadraticTimeMMD& owner;
-	unique_ptr<CMultiKernelQuadraticTimeMMD> multi_kernel;
+	shared_ptr<CMultiKernelQuadraticTimeMMD> multi_kernel;
```

Comment:
meta examples now use `some` so cannot have a unique ptr inside shogun and expose the raw pointer via SWIG

Reply:
but this way you loose:
 1) std::move the thing around - maybe not so important
 2) enforce sole ownership of the resource.... @lambday was this intentional to have it as unique? i would presume yes....

Reply:
discussed with him, he was fine with this change. Alternative would be to always return a new instance in the `::multiclass()` method (which currently returns the raw pointer)